docs/build/identity.rs/1.0/docs/references/specifications/iota-did-method-spec.mdx

```diff
@@ -28,7 +28,7 @@ Data types and subschemas used throughout this TIP are defined in [TIP-21](https
 
 ### UTXO Ledger
 
-The unspent transaction output ([UTXO](/introduction/reference/details#unspent-transaction-output-utxo)) model defines a ledger state which is comprised of unspent outputs. Outputs are created by a transaction consuming outputs of previous transactions as inputs. The Stardust version of the protocol defines several output types, the relevant ones for the IOTA DID Method are: Basic Outputs for _value transactions_, and Alias Outputs for storage of DID Documents.
+The unspent transaction output ([UTXO](/learn/protocols/chrysalis/core-concepts/switch-to-UTXO/)) model defines a ledger state which is comprised of unspent outputs. Outputs are created by a transaction consuming outputs of previous transactions as inputs. The Stardust version of the protocol defines several output types, the relevant ones for the IOTA DID Method are: Basic Outputs for _value transactions_, and Alias Outputs for storage of DID Documents.
```

Comment:
Is there a better source in Stardust? I feel the linked document should be deprecated at some point.

Reply:
It will eventually, we could use the tip if you prefer

```suggestion
The unspent transaction output ([UTXO](/tips/tips/TIP-0020/)) model defines a ledger state which is comprised of unspent outputs. Outputs are created by a transaction consuming outputs of previous transactions as inputs. The Stardust version of the protocol defines several output types, the relevant ones for the IOTA DID Method are: Basic Outputs for _value transactions_, and Alias Outputs for storage of DID Documents.
```